include/camp/concepts.hpp

```diff
@@ -167,6 +167,9 @@ namespace concepts
   template <typename... Args>
   using enable_if = typename std::enable_if<all_of<Args...>::value, void>::type;
 
+  template <typename T, typename... Args>
```

Comment:
You might want to implement enable_if using enable_if_t now?

Reply:
This implementation looks good, but I'm not sure if I want to take it right now rather than reworking it.  I want to have this, but our current definition of enable_if is already a bit brain-bending based on the standard version, and this one takes that and makes it backwards... It's on my list to rectify this and various other naming and interface inconsistency issues, and it's highly related to the concepts rework material.  I guess I'm fine with it as long as we try not to spread it too far to minimize the breakage when we come back to fix this stuff.